lib/large_object_store.rb

```diff
@@ -14,8 +14,11 @@ module LargeObjectStore
   RAW = 2
   FLAG_RADIX = 32 # we can store 32 different states
 
-  def self.wrap(store, **options)
-    RailsWrapper.new(store, options)
+  def self.wrap(*args)
+    RailsWrapper.new(*args)
+  end
+  class << self
+    ruby2_keywords :wrap if respond_to?(:ruby2_keywords, true)
```

Comment:
FWIW i've just been including the `ruby2_keywords` gem as dependencies when I need to do this and avoiding the need for `if respond_to?(:ruby2_keywords)`.

The gem just defines `ruby2_keywords` as a no-op if it's not provided by the VM.